CHANGES.md

```diff
@@ -1,5 +1,15 @@
 ## DC/OS 1.13.0
 
+### Highlights
+
+#### Introduction of service accounts, alignment of authentication architectures
+
+The core of the DC/OS Enterprise identity and access management service (IAM) has been open-sourced and added to DC/OS, replacing `dcos-oauth`. CockroachDB was added as a DC/OS component as a highly available database serving the IAM.
+
+With that DC/OS now supports service accounts. Service accounts are meant for giving an identity to individual tools and applications which are interacting with a DC/OS cluster. They provide a canonical way to log in and obtain authentication proof -- the DC/OS authentication token. Presenting a valid DC/OS authentication token is required for passing access control in Master Admin Router.
```

Comment:
Some of these sentences sound a bit wrong. Maybe consider these alternatives?

Second sentence: `Service accounts allow individual tools and applications to interact with a DC/OS cluster using their own identity.`

I'm not sure that "canonical" is the correct word here. That suggests that service account login is somehow more fundamental than user account login. If anything, "canonical" belongs later: "obtain canonical authentication proof" to associate it with the authentication token rather than the service account.

Final sentence: `A valid DC/OS authentication token is required in order to access DC/OS services through Master Admin Router.`

Reply:
Diff:

```
-With that DC/OS now supports service accounts. Service accounts are meant for giving an identity to individual tools and applications which are interacting with a DC/OS cluster. They provide a canonical way to log in and obtain authentication proof -- the DC/OS authentication token. Presenting a valid DC/OS authentication token is required for passing access control in Master Admin Router.

+With that DC/OS now supports service accounts. Service accounts allow individual tools and applications to interact with a DC/OS cluster using their own identity. A successful service account login results in authentication proof -- the DC/OS authentication token. A valid DC/OS authentication token is required in order to access DC/OS services and components through Master Admin Router.
```

Squashed, will force-push.